velox/exec/Exchange.cpp

```diff
@@ -181,6 +206,29 @@ std::unique_ptr<ExchangeSource> createLocalExchangeSource(
 
 } // namespace
 
+uint64_t ExchangeQueue::expectedSerializedPageSize() const {
+  // Initially, max size / number of sources. Later, average of received sizes.
+  constexpr int32_t kMinExpectedSize = PartitionedOutput::kMinDestinationSize;
```

Comment:
It would be nice to avoid coupling ExchangeQueue with PartitionedOutput operator.